sdk/tests/conformance/ogles/ogles-utils.js

```diff
@@ -722,7 +722,7 @@ function runBuildTest(test, callback) {
       compileSuccess = (success[0] && success[1]);
       if (!test.compstat) {
         if (compileSuccess) {
-          testFailed("expected compile failure but was successful");
+          testPassed("expected compile failure but was successful (spec doesn't require reporting errors at compile time)");
```

Comment:
I haven't looked at these tests in detail but this looks like we'll lose coverage. If compstat is false then we'll never check the link status, and we want to at least make sure that for shaders with compstat=false, we confirm that they'd never link.

For the tests which set compstat=false, do they have matching shaders which would allow a link to be attempted?

Reply:
My bad for being dizzy here.